### ISSUE TITLE
Heavily Modded Crash

### ISSUE THREAD

**Issue by Kagenokami** [TheTemportalist/Compression#15]
My game is heavily modded and I attempted to add in the compression mod to make the game a little simpler via additional storage capabilities. However, when I attempt to start the game I now get a crash. Can anyone figure out what is causing this? I, unfortunately, am unable to understand the log.

http://pastebin.com/E7fTKfSc

**Comment by AetheriumAeon** [comment]
While I don't really understand error logs either, I do know that I was having a similar issue with the mod. It turned out to be an incompatibility issue with the mod 'Adventure Backpacks'. If you have that mod installed I'd recommend disabling that mod's jar file (add '.disabled' to the end of the jar name, after the .jar extension) and trying to run the game then.

If you don't have 'Adventure Backpacks' or the error still happens, all I can suggest is disabling mods one by one until you identify the one where the incompatibility is.

**Comment by TheTemportalist** [comment]
It is likely there is a block or item that is causing compression to error out. Next time I get a chance, I'll take a look at the compression side of things. If you add the full minecraft log to this issue, I'll be able to more quickly diagnose what is causing the error.

**Comment by TheTemportalist** [comment]
You're build of RFTools is WAY too old for me to find in https://github.com/McJty/RFTools
You are going to need to submit this bug to RFTools before we can figure out what is going on here (they have done a lot of refactoring since this version of the mod was built).
If you have already done that, please link the issue on the RFTools tracker so that I can read that. Otherwise, please feel free to come back to this issue when you have more information!

**Comment by TheTemportalist** [comment]
Also, you are using a much older version of Compression. Correct me if I am wrong, but this is still scala code (in the log you will see it says "CBlocks.scala"). The last version of scala code I used was like 1.8/1.9ish. The mod is updated for 1.12, and you should use that.
